integration/management-api-tests/src/test/java/org/wso2/micro/integrator/api/TemplatesResourceTestCase.java

```diff
@@ -19,17 +19,13 @@
 package org.wso2.micro.integrator.api;
 
 import org.json.JSONObject;
-import org.testng.Assert;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
 
 import java.io.IOException;
 
-import static org.wso2.micro.integrator.api.Constants.COUNT;
-import static org.wso2.micro.integrator.api.Constants.LIST;
-
 public class TemplatesResourceTestCase extends ESBIntegrationTest {
```

Comment:
We can move the `@BeforeClass` method to the same abstract class as in [1]

[1] - https://github.com/wso2/micro-integrator/pull/2821/files#r1051774802

Reply:
keeping the before class without moving since the endpoint URL varies for each test case and it needs to be defined in a beforemethod since it's  repeatedly used in several functions in each class

Reply:
Since only the resource path is changing, let's pass the resource path to the sendHttpRequestAndGetPayload method and concat it to the management API endpoint

Reply:
fixed